cmd/kubelet/app/server_test.go

```diff
@@ -222,12 +232,15 @@ clusterDNS:
 			kubeletConfig := &kubeletconfiginternal.KubeletConfiguration{}
 			kubeletFlags := &options.KubeletFlags{}
 
-			if len(test.kubeletConfig) > 0 {
+			if !reflect.DeepEqual(test.kubeletConfig, kubeletConfig) {
```

Comment:
TBH this seems kind of expensive for a check that's quite likely to be the case. Am I missing a reason to keep this?

Reply:
Actually, I changed the data type for that field to `*kubeletconfiginternal.KubeletConfiguration`

Reply:
Fixed